tests/tests.rs

```diff
@@ -272,6 +272,18 @@ fn dynamic_components() {
     );
 }
 
+#[test]
+fn spawn_buffered_entity() {
+    let mut world = World::new();
+    let mut buffer = CommandBuffer::new();
+    let ent = world.reserve_entity();
+
+    buffer.spawn_at(ent, (1, true));
+    buffer.run_on(&mut world);
+
+    assert!(world.contains(ent));
```

Comment:
This test should verify that the components have the correct values, and should spawn multiple entities having some distinct component types and some of the same type with different values.

Reply:
I hope new test covers it all